### DISCUSSION TITLE
Error: Request failed... `"required_login":true`... (code:0xf1)

### DISCUSSION THREAD

**Discussion by zxcvryo** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#67] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88883913/205287686-604f602c-c9c5-4f30-8242-8cb7ccce4cce.png)

I had to change pw because of phishing error from IG, is it because of that?

**Comment by chriskyfung** [comment, marked as answer]
You overused the script and made Instagram consider you as a bot.
